x-pack/platform/plugins/shared/triggers_actions_ui/kibana.jsonc

```diff
@@ -26,7 +26,6 @@
       "dataViewEditor",
       "alerting",
       "actions",
-      "dashboard",
```

Comment:
@dominiqueclarke Are we sure about this? I checked when this dependency was introduced and looks like it was introduced [here](https://github.com/elastic/kibana/pull/159075)

Reply:
I checked with the Triggers Action team to confirm it's not in use. It was previously in use, but then some refactoring happened that removed the need for the dependency, but they forgot to remove it.

Reply:
That's good news, because I faced a cycle dependency issue when I tried to add the cases plugin as a dependency of the dashboard plugin as part of this [issue](https://github.com/elastic/kibana/issues/222616#issuecomment-2988597656).